tests/unit/test_cv_device_tools.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
```

Comment:
Do we need this to be executable?

Have a read through this: https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take#:~:text=The%20shebang%20line%20in%20any,manager%20(when%20configured%20properly).

Though in this case I would just remove it - we don't want the script to be executed but rather pytest to execute the tests